### PR TITLE
Log network address of DNS clients

### LINKED ISSUE #1163: Log the IP address of the machine making DNS requests
Currently the Consul logs display something like this:

```
2015/08/09 02:39:34 [ERR] dns: all resolvers failed for {www.google.com. 1 1}
```

It would be great if that error + the debug logs included the IP address of the query itself:

```
2015/08/09 02:39:34 [ERR] dns: all resolvers failed for {www.google.com. 1 1} - 1.2.3.4
```

### PR DESCRIPTION
**Pull request by ryanuber**
Fixes #1163

### CODE REVIEW

File: command/agent/dns.go [hunk 1]
```diff
@@ -628,7 +632,9 @@ func (d *DNSServer) handleRecurse(resp dns.ResponseWriter, req *dns.Msg) {
 	q := req.Question[0]
 	network := "udp"
 	defer func(s time.Time) {
-		d.logger.Printf("[DEBUG] dns: request for %v (%s) (%v)", q, network, time.Now().Sub(s))
+		d.logger.Printf("[DEBUG] dns: request for %v (%s) from client %s (%s)",
```
Comment (ryanbreen) [inline command/agent/dns.go]: Looks like you're expecting to interpolate 4 terms and are passing in 5.  I think the pattern for time is missing.

Comment (ryanuber) [inline command/agent/dns.go]: Ah you're right, good catch!

**Comment by slackpad** [comment]
LGTM

**Comment by ryanuber** [comment]
Oops, forgot to merge this. Thanks!

**Comment by sethvargo** [comment]
:heart:

**Pull request merged**